docs/html/cli/pip_wheel.rst

```diff
@@ -30,6 +30,15 @@ Description
 
 This is now covered in :doc:`../reference/build-system/index`.
 
+Differences to `build`
+----------------------
+
+`build <https://pypi.org/project/build/>`_ is a simple tool which can among other things build
+wheels for projects using PEP 517
+``pip wheel`` can do the same but also supports projects not using PEP 517.
```

Comment:
One other significant difference is that `pip wheel` won't build a wheel if it's already available (on PyPI, or another configured index). Also `pip wheel` will download and build from sdists, where `build` requires you to do the downloading and unpacking of the sdist.

So really, `build` only offers the functionality of `pip wheel --no-deps .` whereas `pip wheel has a lot more capabilities.

Reply:
I simplified the text to not include any additional feature